### PR TITLE
Github workflow: trigger unit tests

### PR DESCRIPTION
**Pull request by roypaulin**
Each time there is a push or PR on the main branch, actions are triggered and unit tests are run.

### CODE REVIEW

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
```
Comment (spilchen) [inline .github/workflows/unittests.yml:19]: Could we use the same version that's included in DEVELOPER.md -- 1.16.2?

File: .github/workflows/unittests.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Operator Unit tests
+
+on:
```
Comment (spilchen) [inline .github/workflows/unittests.yml:3]: Can this just be `on: [push, pull_request]`?  The unit tests are quick, so it should get run on any push or PR.

**Review by spilchen — COMMENTED** [review]
Looks good.  Just a question on whether we can do this for each push/PR.

File: .github/workflows/unittests.yml (hunk 1 above)
Comment (roypaulin) [reply .github/workflows/unittests.yml:19]: sure.

**Review by spilchen — APPROVED** [review]

**Pull request merged**